README.md

```diff
@@ -95,6 +99,38 @@ println(best)
 
 For more examples, please see [the unit tests](test/fmin/points.jl).
 
+
+### Ask/Tell API
+TreeParzen.jl also supports tuning via an `ask` and `tell!` interface, where the user is afforded
+a lot of control on what they can do and just need to ask the optimiser for suggestions, and tell
+it about the results.
+
+This allows users to do advanced things such as wrapping up objectives in
+a more complex way, using callbacks, controlling termination, optimising after N suggestions,
+continuing iterating if solution is not satisfactory, and so on.
+
+A basic example:
+```julia
+using TreeParzen
+config = TreeParzen.Config()
+trialhist = TreeParzen.Trials.Trial[]
+
+space = Dict(:x => HP.Uniform(:x, -5., 5.))
```

Comment:
tiny thing I guess just little for consistency. there's `using TreeParzen` and then `config = TreeParzen.Config()` which can be just `config = Config()`, seeing that we then skip `TreeParzen.` later on in this example.

Reply:
Yes I didn't change anything else, just `TreeParzen.HP.*` to `HP.*`

Reply:
Oh, this is readme. ok

Reply:
b99ad2c